src/command_modules/azure-cli-batch/setup.py

```diff
@@ -15,7 +15,7 @@
     logger.warn("Wheel is not available, disabling bdist_wheel hook")
     cmdclass = {}
 
-VERSION = "3.0.3+dev"
+VERSION = "3.1.0+dev"
```

Comment:
Ah, no worries - as long as the automated process maintains the minor version number if patch release version doesn't make too much difference.